src/tests/helpers.js

```diff
@@ -1692,6 +1690,126 @@ export const schemaFieldsetScopedCondition = {
   type: 'object',
 };
 
+export const schemaWithConditionalToFieldset = {
+  additionalProperties: false,
+  type: 'object',
+  properties: {
+    work_hours_per_week: {
+      title: 'Hours per week',
+      type: 'string',
```

Comment:
one thing i didnt spot initially, this should probably be a `number`

Reply:
uff, good catch. fixing it. we definitely need linters for these json schemas.

Reply:
done [fc00b24](https://github.com/remoteoss/json-schema-form/pull/23/commits/fc00b243fa9c0f910260d6ec10031b6bdf51e9b1)